update-wing.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+for dir in */ ; do
+    if [ -f "$dir/package.json" ]; then
+        cd "$dir"        
+        if grep -q "\"winglang\"" package.json; then
+            echo "Updating winglang in '$dir' winglib..."
+            npm update winglang
+        else
+            echo "winglang not found in '$dir' winglib."
+        fi        
+        cd ..
```

Comment:
`cd -` seems safer to me, in case we change the directory layout.

Reply:
yep 👍